src/lib/swapper/swappers/LifiSwapper/utils/lifiChainIdtoChainId/lifiChainIdToChainId.ts

```diff
@@ -0,0 +1,9 @@
+import type { ChainId, ChainReference } from '@shapeshiftoss/caip'
+import { CHAIN_NAMESPACE, toChainId } from '@shapeshiftoss/caip'
+
+export const lifiChainIdToChainId = (lifiChainId: number): ChainId => {
```

Comment:
I wonder if we should run the old `assertIsChainReference()` type assertion over this to make sure it's valid?

Reply:
We can ignore this for now, we'll need to pick it up again when/if we want to support multi-hops.